src/Analytics/Adapter.php

```diff
@@ -14,6 +14,8 @@
 
 namespace Utopia\Analytics;
 
+use Exception;
```

Comment:
We can avoid this by doing `new \Exception()` instead of `new Exception()`. IIRC, it's a bit faster. We do it in Appwirte.